src/chrome/content/rules/HurricaneElectric.xml

```diff
@@ -1,43 +1,27 @@
 <!--
 	Other Hurricane Electric rulesets:
-
 		- TunnelBroker.xml
 
-
-
-	Nonfunctional subdomains:
-
-		- bgp.he.net	(refused)
-		- man.he.net	(redirects to www.he.net, valid cert)
-
-
-	bgp.he.net and faq.he.net don't support SSL at the moment
-
-
-	Problematic domains:
-
-		- (www.)hu.com *
-		- hurricaneelectric.com *
-
-	* Alternative domains, same content, using he.net certificate
-
+	www.hurricaneelectric.net does not exist.
 -->
-<ruleset name="Hurricane Electric">
-
+<ruleset name="Hurricane Electric (partial)">
 	<target host="he.com" />
 	<target host="www.he.com" />
-	<target host="he.net" />
-	<target host="*.he.net" />
-	<target host="hurricaneelectric.net" />
-
 
-	<securecookie host="^(?:.*\.)?he\.net$" name=".+" />
-
-
-	<rule from="^http://(?:www\.)?h(?:e\.com|urricaneelectric\.net)/"
-		to="https://www.he.net/" />
+	<target host="he.net" />
+	<target host="www.he.net" />
+	<target host="admin.he.net" />
+	<target host="bgp.he.net" />
+	<target host="code.he.net" />
+	<target host="cust.he.net" />
+	<target host="dns.he.net" />
+	<target host="ipv6.he.net" />
 
-	<rule from="^http://((?:admin|code|cust|dns|ipv6|www)\.)?he\.net/"
-		to="https://$1he.net/" />
+	<target host="hurricaneelectric.net" />
 
+	<securecookie host=".+" name=".+" />
+	
+	<rule from="^http://(www\.)?he\.com/" to="https://$1he.net/" />
```

Comment:
This rule can be for `www` only, `^` is already covered by the generic rule below.

Reply:
The redirect is for `.com` → `.net`.

Reply:
Ah, therefore the comment for `.com` should be kept